### ISSUE TITLE
Update Mavericks and Yosemite guide to use system ruby

### ISSUE THREAD

**Issue by lengarvey** [reinteractive-open/installfest_guides#56]
We can get away without installing chruby etc if we just simply use the system ruby. The "best" way would probably be to install the compiler chain (and maybe homebrew).

1. Document the appropriate environment variables required to setup system ruby to install gems without `sudo`
2. Move the "advanced" guide elsewhere since it's definitely useful but slower than we'd like for installfest.
3. Consider contributing to the RailsGirls guide on the same topic.

**Comment by Zorlin** [comment]
I disagree - the system ruby is not as up-to-date, and is updated much more rarely. Chruby also allows for local gems (no sudo) much more easily.
